spec/user_interface_spec.rb

```diff
@@ -160,7 +162,7 @@
 
     it 'ignores case sensitivity on valid inputs' do
```

Comment:
I would move this test to the validator, as case sensitivity is now knowledge that belongs to the validator.

Reply:
? 
the constant is in lowercase